reading/ecto.livemd

```diff
@@ -148,14 +148,14 @@ config :journal, Journal.Repo,
 Now create the local database.
 
 ```
-$ mix ecto.create
+mix ecto.create
 The database for Journal.Repo has been created
 ```
 
 Similar to the `createdb` command used in the RDBMS lesson, this creates a `journal` database in PostgreSQL. We can verify this from the `psql` prompt.
```

Comment:
Should this be `journal_repo` as well?

Reply:
https://github.com/DockYard-Academy/beta_curriculum/pull/309